tests/test_boundary_layer.py

```diff
@@ -23,10 +22,11 @@ def test_mixed_layer_height_RHmax(ds_isentropic_test_profiles):
     )
     assert np.allclose(da_rh_peak, z0)
 
-
-def test_mixed_layer_height_RH_lin():
-    ds = eurec4a_environment.source_data.open_joanne_dataset()
-    ds = ds.isel(sounding=slice(0, 10))
-    da_lin = boundary_layer.mixed_layer_height.calc_peakRH_linearize(ds)
-    assert len(da_lin) == 10
-    assert np.all(da_lin < 1500.)
+def test_inversion_height_gradient_RH(ds_isentropic_test_profiles):
+    ds = ds_isentropic_test_profiles.copy()
+    z_INV = 2000.
+    ds['RH'] = ds.RH.where(ds.alt < z_INV, other=0.5)
+    print (ds.RH)
+    da_inv = boundary_layer.inversion_height.find_inversion_height_grad_RH(ds=ds, rh = 'RH')
+    print (da_inv)
```

Comment:
Love the simplicity of this test!  😄  Did you meant to leave the `print` statements in though?

Reply:
Ups no